cuda_core/cuda/core/experimental/_module.py

```diff
@@ -46,26 +73,27 @@ def __init__(self, module, code_type, jit_options=None, *,
                  symbol_mapping=None):
         if code_type not in self._supported_code_type:
             raise ValueError
+        _lazy_init()
         self._handle = None
 
-        driver_ver = handle_return(cuda.cuDriverGetVersion())
-        self._loader = _backend["new"] if driver_ver >= 12000 else _backend["old"]
+        backend = "new" if (_py_major_ver >= 12 and _driver_ver >= 12000) else "old"
+        self._loader = _backend[backend]
 
         if isinstance(module, str):
-            if driver_ver < 12000 and jit_options is not None:
+            if _driver_ver < 12000 and jit_options is not None:
                 raise ValueError
             module = module.encode()
             self._handle = handle_return(self._loader["file"](module))
```

Comment:
Looks like jit_options are ignored for 12.x. Is it a TODO for this feature?

Reply:
Unfortunately as of today it's not possible to use this feature without also building the bindings from source (https://github.com/NVIDIA/cuda-python/issues/73). Let's just disable this path entirely for the time being.

Reply:
Skipped in commit 74de685475cdc1c0d114de10552a94e9de88f53b.

Reply:
Tracked in https://github.com/NVIDIA/cuda-python/issues/194.

Reply:
(btw @vzhurba01 thanks for spotting this 👍 it would have been a bug if we had caught it!)